libmisc/pwd_init.c

```diff
@@ -31,32 +31,32 @@ void pwd_init (void)
 #ifdef HAVE_SYS_RESOURCE_H
 	struct rlimit rlim;
 
-#ifdef RLIMIT_CORE
+# ifdef RLIMIT_CORE
```

Comment:
I prefer to indent all the line and not just the code after `#`. @hallyn any opinion?

Reply:
Being able to grep for things like `^# *define` is a nice feature.  Not having the `^` can add some noise.

Reply:
I personally find `#` ugly in non-first columns.  Having it in the first column keeps it separate from readable code.  But it's my personal taste, so I don't want to enforce it.  Are there any other opinions?

Reply:
Anyway, it seems I burnt that line in a following commit :)

Reply:
Sorry I'm not sure what the conversation is about here - but in general I like to see #ifdefs start at column 0...

I *prefer* to have ifdefs when possible be avoided using careful code refactoring :)  (section "21) Conditional Compilation" at https://www.kernel.org/doc/html/latest/process/coding-style.html).

Reply:
> Sorry I'm not sure what the conversation is about here - but in general I like to see #ifdefs start at column 0...
> 

So, I added some indentation to ifdefs to distinguish nested ones.  I prefer:

```c
#if A
# if B
...
# endif
#endif
```

While, Iker prefers to indent the `#` (I don't know by how much; a space?  a tab?)

```c
#if A
 #if B
...
 #endif
#endif
```

> I _prefer_ to have ifdefs when possible be avoided using careful code refactoring :) (section "21) Conditional Compilation" at https://www.kernel.org/doc/html/latest/process/coding-style.html).

Oh, sure, I hate cpp(1) stuff as much as anyone :)

Reply:
Yup I don't expect refactoring as part of other work :)

Let's stick with the pre-existing style.  I mean personally I think in projects I've done neither type of indentation - think of the maintenance cost when an ifdef is added or removed from the nesting stack :)

Reply:
> Yup I don't expect refactoring as part of other work :)
> 
> Let's stick with the pre-existing style. I mean personally I think in projects I've done neither type of indentation - think of the maintenance cost when an ifdef is added or removed from the nesting stack :)

What then?  No indentation at all?  Heh, cpp is ugly, but without indentation, it's really unreadable (to me) when nested :)

Reply:
So, I guess that's all; the patch set is ready.  I can open a few issues to discuss the side questions that we have, but they don't affect this cleanup.  Could you please have a look at it?